test/classes/ferguson/delete-free/owned-demo.chpl

```diff
@@ -56,28 +56,28 @@ proc examples() {
   var owned5 = new owned C(5)?;
   writeln("owned5 = ", owned5.borrow());
 
-  // The clear method can be used to delete the
+  // The release method can be used to delete the
```

Comment:
Note that this code doesn't use the release method anymore so the comment is out-of-date.